lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -178,7 +178,7 @@ def send_request(packet, timeout = self.response_timeout)
     if timeout.nil?
       return nil
     elsif response.nil?
-      raise Rex::TimeoutError.new("Send timed out")
+      raise Rex::TimeoutError.new("Send timed out. Timeout currently #{timeout} seconds, you can configure this with %grnsessions --interact <id> --timeout <value>%clr")
```

Comment:
I think this error message change is too low down in the stack, i.e. this new error message could trigger when a post modules timeout

The error message should only occur when the module was being interacted with and a timeout exception is raised :+1:

Reply:
I'll get back at it 👍